### PR TITLE
Get removal round 2

### PR DESCRIPTION
**Pull request by fengalin**
## Note to applications developers

Applications developers should use [`fix-getters-calls` 0.3.0](https://crates.io/crates/fix-getters-calls/0.3.0) to ease migration of their applications. If you also wish to change your get functions definition to comply with the standards implemented here, use [`fix-getters-def` 0.3.0](https://crates.io/crates/fix-getters-def/0.3.0).

## Object

This PR addresses two concerns identified during https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/merge_requests/639:

- `enum` getters generated automatically required specific renames in `Gir.toml` because they were consuming `self`.
- `get` elision was not applied in the `subclass` infrastructure.

This second round addresses other important concerns. See the details in this [gir PR](https://github.com/gtk-rs/gir/pull/1102). Also depends on https://github.com/gtk-rs/gtk-rs/pull/455.

### CODE REVIEW

File: gsk4/Gir.toml [hunk 1]
```diff
@@ -90,6 +90,10 @@ cfg_condition = "feature = \"broadway\""
 name = "Gsk.GLShader"
 status = "generate"
 generate_builder = true
+    [[object.function]]
+    name = "get_arg_bool"
```
Comment (bilelmoussaoui) [inline gsk4/Gir.toml:94]: Why this one needs an explicit rename but not the other ones such as arg_int?

File: gtk4/src/auto/stack_page.rs [hunk 2]
```diff
@@ -42,7 +42,7 @@ impl StackPage {
     }
 
     #[doc(alias = "gtk_stack_page_get_needs_attention")]
-    pub fn is_needs_attention(&self) -> bool {
+    pub fn needs_attention(&self) -> bool {
```
Comment (bilelmoussaoui) [inline gtk4/src/auto/stack_page.rs:45]: Nice! :) i really like the new name

Comment (fengalin) [reply gtk4/src/auto/stack_page.rs:45]: Thanks :)

Comment (fengalin) [reply gtk4/src/auto/stack_page.rs:45]: BTW, this is an automatic conversion on behalf of `fix-getters-rules` which is also automatically detected on the call sites when migrating code using `fix-getters-calls`.

File: gsk4/Gir.toml (hunk 1 above)
Comment (fengalin) [reply gsk4/Gir.toml:94]: What I meant is that the return value is not an answer to "is this arg a boolean?", but the actual value of the `arg`.

Comment (bilelmoussaoui) [reply gsk4/Gir.toml:94]: Ah I see. Makes sense.

**Pull request merged**